elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -374,6 +374,41 @@ class RestlasticSearchClientTest extends WordSpec with Matchers with ScalaFuture
       }
     }
 
+    "Support bulk partial document updates with whole document upserts" in {
```

Comment:
Does this test need 4 documents, or can it be shortened to use fewer, may be 2, documents? That would make it easier to understand.

Reply:
Yeah, good point. Reducing to 2 documents

Reply:
test updated here https://github.com/SumoLogic/elasticsearch-client/pull/151/commits/bb0633cae40cbdf5f0866424878f23325c21d391